pkg/k8shandler/cluster.go

```diff
@@ -136,6 +136,13 @@ func (elasticsearchRequest *ElasticsearchRequest) CreateOrUpdateElasticsearchClu
 					elasticsearchRequest.cluster.Namespace,
 					elasticsearchRequest.client,
 					int32(calculateReplicaCount(elasticsearchRequest.cluster)))
+
+				// add alias to old indices if they exist and don't have one
+				// this should be removed after one release...
+				AddAliasForOldIndices(
```

Comment:
I want to short circuit this -- if it returns `true` we should skip this for future iterations since we would no longer be creating old indices